appveyor-ubuntu.yml

```diff
@@ -44,11 +44,11 @@ install:
   # AppVeyor's apt-get cache might be outdated, and the package could potentially be 404.
   - sh: "sudo apt-get update --allow-releaseinfo-change"
 
-  # install coretto 21
+  # install coretto 25
```

Comment:
So are we *not* installing Java21? (is it installed already or do we not need it?)

Reply:
We need Java 25 to run the Java related tests. GitHub Actions already use it.